### ISSUE TITLE
Configuring script: "javac: invalid flag -c"

### ISSUE THREAD

**Issue by andyrichardson** [atom-community/atom-script#1106]
I'm trying to execute the following script inside of atom but it produces an error.

```
 javac -cp . labs/shapes/Shape.java; java labs/shapes/Shape
```
### Replication

I use the configure run options tool and set the following parameters

**Command:** `javac`
**Command Arguments:** `-cp . {FILE_ACTIVE}`

And then hit run. This is the resulting output.

```
javac: invalid flag: -c
Usage: javac <options> <source files>
use -help for a list of possible options
```

<hr>

If I change the _Command Arguments_ parameter to nothing, the same error occurs.

**Comment by aminya** [comment]
Should be fixed in #2470. Let me know if it is not.

**Comment by Patrick-Polgar** [comment]
I have the same issue... what is the solution?
